### ISSUE TITLE
Update metric query API to support usage.stats.initial_release_date using dynamoDB

### ISSUE THREAD

**Issue by manasaV3** [chanzuckerberg/napari-hub#857]

**Comment by manasaV3** [comment]
Blocked by #882

**Comment by richaagarwal** [comment]
Currently, we retrieve this information on the frontend by directly accessing the existing plugin field `first_released`. Since we already load this data on the plugin page, as a part of https://github.com/chanzuckerberg/napari-hub/blob/main/frontend/src/context/plugin.tsx, I'm curious what the value would be of transitioning over a single data point without taking into consideration the others.  cc @manasaV3 @codemonkey800

**Comment by manasaV3** [comment]
Making just this data point available in metrics endpoint because the first_released date is being used in the activity dashboard to generate the time frame in `x installs since publicly released over y months ago`.

**Comment by richaagarwal** [comment]
After discussing with @manasaV3 , this is not critical to the migration effort but a nice to have, so I will be adjusting the epic accordingly
